SwiftWisdom/Core/UIKit/UIView+Extensions.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UIView+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Alexander Persian on 12/29/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+
+    /// Zooms a view by scaling using the provided factor.
+    func ip_zoom(by factor: CGFloat) {
+        transform = CGAffineTransform(scaleX: factor, y: factor)
```

Comment:
Those two are different things.
`CGAffineTransform(scaleX:y:)` creates a new scale transform.  The method as it stands will actually remove any other transformation applied.
`transform.scaledBy(x:y:)` applies it to the current transform